src/signals/settings/users/containers/Overview/__tests__/Overview.test.js

```diff
@@ -440,4 +422,176 @@ describe('signals/settings/users/containers/Overview', () => {
 
     expect(dispatch).toHaveBeenCalledTimes(1);
   });
+
+  it('should render a role filter', async () => {
+    const { getByTestId } = render(usersOverviewWithAppContext());
+
+    await wait();
+
+    expect(getByTestId('roleSelect')).toBeInTheDocument();
+  });
+
+  it('should select the right option when role filter changes', async () => {
+    jest
+      .spyOn(appSelectors, 'makeSelectUserCan')
+      .mockImplementation(() => () => true);
+
+    jest
+      .spyOn(rolesSelectors, 'inputRolesSelector')
+      .mockImplementation(() => inputRolesSelectorJSON);
+
+    const { getByTestId } = render(usersOverviewWithAppContext());
+
+    await wait();
+
+    const filterByRoleSelect = getByTestId('roleSelect');
+
+    // check if the default value has been set
+    expect(filterByRoleSelect.value).toBe('*');
+
+    const filterValue = 'Regievoerder';
+
+    act(() => {
+      fireEvent.change(filterByRoleSelect, { target: { value: filterValue } });
+    });
+
+    expect(dispatch).toHaveBeenCalledTimes(1);
+    expect(dispatch).toHaveBeenCalledWith(setUserFilters({ role: filterValue }));
+
+    expect(filterByRoleSelect.value).toBe(filterValue);
+
+    const activeOption = filterByRoleSelect.querySelector('select option:nth-child(8)');
+
+    expect(activeOption.value).toBe(filterValue);
+  });
+
+  it('should render a user active (status) filter', async () => {
+    const { getByTestId } = render(usersOverviewWithAppContext());
+
+    await wait();
+
+    expect(getByTestId('userActiveSelect')).toBeInTheDocument();
+  });
+
+  it('should select the right option when user active (status) filter changes', async () => {
+    const { getByTestId } = render(usersOverviewWithAppContext());
+
+    await wait();
+
+    const filterByUserActiveSelect = getByTestId('userActiveSelect');
+
+    // check if the default value has been set
+    expect(filterByUserActiveSelect.value).toBe('*');
+
+    const filterValue = 'true';
+
+    act(() => {
+      fireEvent.change(filterByUserActiveSelect, { target: { value: filterValue } });
+    });
+
+    expect(dispatch).toHaveBeenCalledTimes(1);
```

Comment:
Same comment here; can you assert that `dispatch` hasn't been called prior to firing the event?